core/domain/exp_domain.py

```diff
@@ -3648,6 +3712,20 @@ def _convert_v23_dict_to_v24_dict(cls, exploration_dict):
 
         return exploration_dict
 
+    @classmethod
+    def _convert_v24_dict_to_v25_dict(cls, exploration_dict):
+        """ Converts a v23 exploration dict into a v24 exploration dict.
```

Comment:
I'm confused--the docstring says v23->v24, but the code looks like it is setting to v25?

Reply:
My bad - updated!